portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+public class Comment{
+    //personal data
+    private String firstName, lastName, email, phone;
+    //job related data: optional
+    private boolean jobRelated=false;
+    private String jobTitle="";
+    //message 
+    private String message;
```

Comment:
It is good practice to mark fields as `final`, when possible (immutability is good). In this case, we could do it for `firstName`, `lastName`, `email`, `phone`, and `message`. 

http://go/ej3e-17

Reply:
done